src/Mod/Fem/App/FemConstraintInitialTemperature.h

```diff
@@ -42,7 +42,8 @@ class FemExport ConstraintInitialTemperature: public Fem::Constraint
 
     // Temperature parameters
     App::PropertyTemperature initialTemperature;
-
+    App::PropertyBool EnableFinalTemperature;
+    App::PropertyTemperature finalTemperature;
```

Comment:
Capitalize the first letter of the property. I know initialTemperature is lowercase, but we should fix that for consistency.